app/controller/grid/ItemDeleterGridControllerMixin.js

```diff
@@ -0,0 +1,45 @@
+Ext.define('CpsiMapview.controller.grid.ItemDeleterGridControllerMixin', {
+    extend: 'Ext.Mixin',
+
+    onDeleteRowClick: function (tableView, rowIndex, colIndex, item, e, record, tableRow) {
+        var me = this;
+        // what to do for deleting
+        var doDelete = function () {
```

Comment:
Not sure I understand the intent here? This function can't be overridden, but does nothing?